opm/parser/eclipse/OpmLog/EclipsePRTLog.cpp

```diff
@@ -0,0 +1,80 @@
+/*
+  Copyright 2016 Statoil ASA.
+
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include <opm/parser/eclipse/OpmLog/EclipsePRTLog.hpp>
+#include <opm/parser/eclipse/OpmLog/LogUtil.hpp>
+
+namespace Opm {
+
+
+    EclipsePRTLog::EclipsePRTLog(const std::string& logFile, int64_t messageMask) : StreamLog(logFile, messageMask)
+    {}
```

Comment:
As long as these constructors only call the base constructor you do not need them?